src/OpenBinary/reader.jl

```diff
@@ -2,10 +2,13 @@ Base.@kwdef mutable struct Reader <: PSRI.AbstractReader
     io::IOStream
 
     # stages
-    stage_total::Int
-    first_year::Int
-    first_stage::Int
     first_relative_stage::Int
```

Comment:
Do we still need the first_relative_stage field?

Reply:
Probably not